src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2410,10 +2410,23 @@ private void OnAutoEvoPredictionComplete(PendingAutoEvoPrediction run)
         autoEvoPredictionRunSuccessful = true;
 
         // Set the initial value
-        totalPopulationLabel.ResetInitialValue();
-        totalPopulationLabel.Value = run.PlayerSpeciesOriginal.Population;
+        totalEnergyLabel.ResetInitialValue();
+        totalEnergyLabel.Value = run.PlayerSpeciesOriginal.Population;
 
-        totalPopulationLabel.Value = newPopulation;
+        results.GetPatchEnergyResults(run.PlayerSpeciesNew).TryGetValue(
+            Editor.CurrentPatch, out var newTotalPatchEnergyGathered);
+        var totalEnergyLabelText = totalEnergyLabel.GetNode<Label>(totalEnergyLabel.ValuePath);
+
+        if (newTotalPatchEnergyGathered != null)
+        {
+            totalEnergyLabel.Value = newTotalPatchEnergyGathered.TotalEnergyGathered;
+            if (totalEnergyLabelText != null)
+            {
+                totalEnergyLabelText.Text =
+                    @$"{StringUtils.ThreeDigitFormat(
+                        newTotalPatchEnergyGathered.TotalEnergyGathered)} ({newPopulation})";
+            }
+        }
```

Comment:
If this if doesn't succeed, then I think an error message should be printed as otherwise the wrong number is displayed in the GUI without further information as to what went wrong.

Reply:
Does the red text that pops up saying something like "prediction failed" has to display?

Also I noticed the icon is somewhat small when prediction fails:
![image](https://github.com/Revolutionary-Games/Thrive/assets/100433934/ef869e6f-eae3-482b-96ac-63bb3184653d)

Reply:
> Also I noticed the icon is somewhat small when prediction fails:

I think it's been like that since it was added. It's visible for such a small amount of time / rarely in case of failure, that I haven't bothered fixing it as it might be a bit tricky to fix it.

Reply:
I meant should I make it display when game can't get the gathered energy. I don't know if it displays when auto-evo fails itself, but I don't think I messed up anything to make it not do so.

Reply:
That would be pretty nice. Otherwise we might get a really confusing bug report in the future about the numbers not making sense (and only a log file being able to clarify the situation).